PublicMapping/districtbuilder
# Feature/mvm/map demographic labels

## Overview

This PR adds a control to the project page to display population / other demographic data as labels on the map.

### Demo

![localhost_3003_projects_31b83fa2-4fcd-4bbb-954f-275bb2059af1](https://user-images.githubusercontent.com/4432106/87068999-718d2080-c1e4-11ea-98e0-79ccc52eb93a.png)


### Notes

 - This is hardcoded to only display the county labels - this should be changed after or as part of #200 to show the current geolevel
 - I had to make a small change to the `process-geojson` script to prevent labels from being dropped at low zoom levels


## Testing Instructions

- Rerun `process-geojson` & `publish-region` and create a new project for that region. You should be able to use the label control to display labels on the map

Closes #189
